xwiki-platform-core/xwiki-platform-rest/xwiki-platform-rest-server/src/main/java/org/xwiki/rest/internal/ModelFactory.java

```diff
@@ -1063,4 +1076,40 @@ public JobLog toRestJobLog(Collection<LogEvent> logs, URI self)
 
         return log;
     }
+
+    /**
+     * check if the given property should be exposed via REST
```

Comment:
Cosmetic: uppercase + dot at the end (will fail checkstyle otherwise).

Reply:
I thought I ran checkstyle, but now I see the class is in the exclude list; that is why I did not notice.